src/hf_hydrodata/model/variable_type.csv

```diff
@@ -8,4 +8,5 @@ run_file,Model run files  (not really varibles but they get stored as catalog en
 stream_network,"Watershed and subbasin IDS, stream order, flow directions, HUCs"
 subsurface,"Watertable depth, soil moisture, soil temperature"
 surface_water,"Streamflow, irrigation, diversions"
-topography,"Elevation (raw and processed), slopes and topographic index"
\ No newline at end of file
+topography,"Elevation (raw and processed), slopes and topographic index"
+"",No variable type
```

Comment:
Same question for all of these? Is there a reason to not just use `null` if a field doesn't apple to a dataset?